### ISSUE TITLE
Change logits entered as arguments not to be destroyed　-repetition_penalty

### ISSUE THREAD

**Issue by qualiteger** [qualiteg/ChatStream#50]
引数として入力されたlogitsを直接変更してしまうコードを修正する
